### PR TITLE
pdftoppm: add page

### PR DESCRIPTION
**Pull request by olgashi**
Issue [7648](https://github.com/tldr-pages/tldr/issues/7648)

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**3.00

Closes: #7648

### CODE REVIEW

File: pages/linux/pdftoppm.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# pdftoppm
+
+> Convert PDF document pages to Portable Pixmap (image formats).
+> More information: <https://linux.die.net/man/1/pdftoppm>.
+
+- Specify the first page to convert:
+
+`pdftoppm -f`
```
Comment (mfrw) [inline pages/linux/pdftoppm.md:8]: Should there be a pdf document which we supply ?
eg like `pdftopm -f {{file.pdf}}`

File: pages/linux/pdftoppm.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# pdftoppm
+
+> Convert PDF document pages to Portable Pixmap (image formats).
+> More information: <https://linux.die.net/man/1/pdftoppm>.
```
Comment (mfrw) [inline pages/linux/pdftoppm.md:4]: We generally prefer `manned` as source for manual pages:
```suggestion
> More information: <https://manned.org/pdftoppm>.
```

**Review by mfrw — CHANGES_REQUESTED** [review]

File: pages/linux/pdftoppm.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Specify the first page to convert:
+
+`pdftoppm -f`
+
+- Specify the last page to convert:
+
+`pdftoppm -l`
```
Comment (mfrw) [inline pages/linux/pdftoppm.md:12]:
```suggestion
`pdftoppm -l {{file.pdf}}`
```

File: pages/linux/pdftoppm.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Generate a monochrome PBM file (instead of a color PPM file):
+
+`pdftoppm -mono`
```
Comment (mfrw) [inline pages/linux/pdftoppm.md:16]:
```suggestion
`pdftoppm -mono {{file.pdf}}`
```

File: pages/linux/pdftoppm.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Generate a grayscale PGM file (instead of a color PPM file):
+
+`pdftoppm -gray`
```
Comment (mfrw) [inline pages/linux/pdftoppm.md:20]:
```suggestion
`pdftoppm -gray {{file.pdf}}`
```

File: pages/linux/pdftoppm.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+- Generate a PNG file instead a PPM file:
+
+`pdftoppm -png`
```
Comment (mfrw) [inline pages/linux/pdftoppm.md:24]:
```suggestion
`pdftoppm -png {{file.pdf}}`
```

**Review by mfrw — COMMENTED** [review]
Thank you for your contribution.
Have a few review comments.

File: pages/linux/pdftoppm.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
+# pdftoppm
+
+> Convert PDF document pages to Portable Pixmap (image formats).
+> More information: <https://manned.org/pdftoppm>.
+
+- Specify the range of pages to convert (N-first page, M-last page):
+
+`pdftoppm -f {{N}} -l {{M}} {{file.pdf}} {{image_name_prefix}}`
```
Comment (blueskyson) [inline pages/linux/pdftoppm.md:8]:
```suggestion
`pdftoppm -f {{N}} -l {{M}} {{path/to/file.pdf}} {{image_name_prefix}}`
```

File: pages/linux/pdftoppm.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+- Convert only the first page:
+
+`pdftoppm -singlefile {{file.pdf}} {{image_name_prefix}}`
```
Comment (blueskyson) [inline pages/linux/pdftoppm.md:12]:
```suggestion
`pdftoppm -singlefile {{path/to/file.pdf}} {{image_name_prefix}}`
```

File: pages/linux/pdftoppm.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+- Generate a monochrome PBM file (instead of a color PPM file):
+
+`pdftoppm -mono {{file.pdf}} {{image_name_prefix}}`
```
Comment (blueskyson) [inline pages/linux/pdftoppm.md:16]:
```suggestion
`pdftoppm -mono {{path/to/file.pdf}} {{image_name_prefix}}`
```

File: pages/linux/pdftoppm.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+
+- Generate a grayscale PGM file (instead of a color PPM file):
+
+`pdftoppm -gray {{file.pdf}} {{image_name_prefix}}`
```
Comment (blueskyson) [inline pages/linux/pdftoppm.md:20]:
```suggestion
`pdftoppm -gray {{path/to/file.pdf}} {{image_name_prefix}}`
```

File: pages/linux/pdftoppm.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+
+- Generate a PNG file instead a PPM file:
+
+`pdftoppm -png {{file.pdf}} {{image_name_prefix}}`
```
Comment (blueskyson) [inline pages/linux/pdftoppm.md:24]:
```suggestion
`pdftoppm -png {{path/to/file.pdf}} {{image_name_prefix}}`
```

**Comment by olgashi** [comment]
@blueskyson thank you for the feedback. I applied the suggested changes.

**Comment by blueskyson** [comment]
Please sign the [CLA license](https://cla-assistant.io/tldr-pages/tldr) so that we can proceed the pull request.

File: pages/linux/pdftoppm.md [hunk 12]
```diff
@@ -0,0 +1,24 @@
+# pdftoppm
+
+> Convert PDF document pages to Portable Pixmap (image formats).
```
Comment (CleanMachine1) [inline pages/linux/pdftoppm.md:3]:
```suggestion
> Convert PDF document pages to portable Pixmap (image formats).
```
should this be capital?

File: pages/linux/pdftoppm.md [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+> More information: <https://manned.org/pdftoppm>.
+
+- Specify the range of pages to convert (N-first page, M-last page):
+
+`pdftoppm -f {{N}} -l {{M}} {{path/to/file.pdf}} {{image_name_prefix}}`
+
+- Convert only the first page:
```
Comment (CleanMachine1) [inline pages/linux/pdftoppm.md:10]:
```suggestion
- Convert only the first page of a PDF:
```

**Comment by olgashi** [comment]
@blueskyson I signed the CLA

**Comment by olgashi** [comment]
@mfrw could you please review the revised changes? Thank you!

**Review by mfrw — APPROVED** [review]
Thank you for the contribution @olgashi 
LGTM with the suggestion from @CleanMachine1

**Comment by olgashi** [comment]
@blueskyson could you please review the changes? thank you!

File: pages/linux/pdftoppm.md (hunk 12 above)
Comment (navarroaxel) [reply pages/linux/pdftoppm.md:3]: @olgashi you didn't answer the question or applied the change.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**